### PR TITLE
openjdk 8.0.412

### PR DESCRIPTION
**Pull request by daniel88ferreira**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

We still need to use java8 for now. This is to add the current latest version.

### CODE REVIEW

**Comment by daniel88ferreira** [comment]
@conda-forge-admin, please rerender

**Comment by daniel88ferreira** [comment]
@h-vetinari osx-64 build has an issue that seems related to code signature. But I see the command is being called to removed it:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=926180&view=logs&j=58ac6aab-c4bc-5de2-2894-98e408cc8ec9&t=933f325c-924e-533d-4d95-e93b5843ce8b&l=1077

So I have no idea of what's going on. Can you please help?

**Comment by h-vetinari** [comment]
Interesting, I've never seen that kind of error before:
```
$BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-install_name_tool: fatal error: file not in an order that can be processed (link edit information does not fill the __LINKEDIT segment): $PREFIX/bin/rmid
```
Perhaps something changed about the way the zulu binaries are packaged that cannot be processed by our tooling?

Any ideas @conda-forge/cctools-and-ld64 @xhochy?

**Comment by h-vetinari** [comment]
If you add the stdlib, you'll need to rerender for things to work. From my side, we can either skip osx-64 here or just merge it despite not passing. Given that osx-arm64 passed (using much of the same machinery), perhaps the next build will work again...?

I'm also not very invested into java 8 anymore, so I'm definitely biased in not wasting too much effort on this. 🤷‍♂️

**Comment by xhochy** [comment]
We're also seeing the `__LINKEDIT` problems over at https://github.com/conda-forge/duckdb-split-feedstock/pull/5 But here, we are building the binaries ourselves.

**Comment by daniel88ferreira** [comment]
@conda-forge-admin, please rerender

**Comment by daniel88ferreira** [comment]
@xhochy do you have any ideas on how can we fix this?

If not, @h-vetinari maybe we can merge this without osx_64

**Comment by daniel88ferreira** [comment]
hello @h-vetinari  and @xhochy can you please unblock me here? :)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
   - url: {{ zulu_url }}/{{ zulu_base }}-linux_aarch64.tar.gz                  # [aarch64]
-    sha256: bde61dd92b47baf9c9a15461c365765ae825b5e6e37602014c457b5be6f7b428  # [aarch64]
-  - url: {{ zulu_url }}/{{ zulu_base }}-macosx_x64.zip                        # [osx and x86_64]
-    sha256: 25bfd1f71562695eda921231fbb2b957d84969afcb814efb299fda85f9d3af51  # [osx and x86_64]
+    sha256: 82c46c65d57e187ef68fdd125ef760eaeb52ebfe1be1a6a251cf5b43cbebc78a  # [aarch64]
+
+  # skipping due to : https://github.com/conda-forge/openjdk-feedstock/pull/162#issuecomment-2091518368
+  # - url: {{ zulu_url }}/{{ zulu_base }}-macosx_x64.tar.gz                     # [osx and x86_64]
+  #   sha256: 2bfa0506196962bddb21a604eaa2b0b39eaf3383d0bdad08bdbe7f42f25d8928  # [osx and x86_64]
+
```
Comment (h-vetinari) [inline recipe/meta.yaml:20]: Please undo the changes here (aside from required sha updates, obviously). The skip for osx below is more than enough

Comment (daniel88ferreira) [reply recipe/meta.yaml]: done! thanks!

**Review by h-vetinari — APPROVED** [review]
Thank you for your patience!

**Comment by daniel88ferreira** [comment]
@h-vetinari is this supposed to be merged automatically now?

**Comment by h-vetinari** [comment]
Yeah, that didn't work as it should

**Comment by daniel88ferreira** [comment]
Thanks!
